README.md

```diff
@@ -30,7 +30,6 @@ The easiest way to get Aardvark up and running is to add a dependency on the Aar
 #### Using [CocoaPods](https://cocoapods.org)
 
 ```
-platform :ios, '8.0'
```

Comment:
should we state our minimum platform version here? Or do we think that folk know how to do this so there's no need?

Reply:
I think generally the platform version specification is going to be set up prior to adding the dependency. Either you already use CocoaPods, in which case you should have it specified already, or you need to set up CocoaPods first, in which case it will be added with the `pod init`. We do list our platform requirements lower down in the README though.